cygnus-ngsi-ld/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.telefonica.iot</groupId>
     <artifactId>cygnus-ngsi-ld</artifactId>
-    <version>2.17.0</version>
+    <version>2.17.0-SNAPSHOT</version>
```

Comment:
L58 should be adjusted in a similar way

Reply:
done in https://github.com/telefonicaid/fiware-cygnus/pull/2148/commits/472a3f5e90defe13c975c4226488077573f1adc9